_docs/release-notes.md

```diff
@@ -24,6 +24,21 @@ Visit [https://app.tidepool.org](https://app.tidepol.org) to see the results of
 
 Don't have a Tidepool account? Visit [tidepool.org/signup](https://tidepool.org/signup) to create your free Tidepool account.  
 
+### 1.12.3 (Released 2018-09-12)
+We added timestamps to CGM data in your Daily View. With so many dots to keep track of, this should help keep things organized.  
+[Feature Requirements](https://trello.com/c/QWz76ymx/16-s-data-viz-daily-view-cgm-add-timestamp-in-hover)
+
+[Code Update](https://github.com/tidepool-org/blip/releases/tag/v1.12.3)
+
+### 1.12.2 (Released 2018-09-11)
+Our new Content Security Policy caused issues for people trying to access Tidepool on Safari. Sorry about that one, that's been fixed.  
```

Comment:
Should we be more specific here to indicate that it's to create an account, as you can't actually "access" Tidepool on Safari?

Reply:
Good thinking. I will be more specific.